internal/graph/lookupsubjects.go

```diff
@@ -460,7 +464,10 @@ func (lsu *lookupSubjectsUnion) CompletedChildOperations() error {
 
 		for _, result := range collector.Results() {
 			metadata = combineResponseMetadata(metadata, result.Metadata)
-			foundSubjects.UnionWith(result.FoundSubjectsByResourceId)
+			err := foundSubjects.UnionWith(result.FoundSubjectsByResourceId)
```

Comment:
You should use the `if err := thing(); err != nil` pattern here and at other callsites.